### DISCUSSION TITLE
Portable Version

### DISCUSSION THREAD

**Discussion by ezequiel9** [beyondcode/herd-community#723] [Ideas]
Herd works great! I would like to have a portable version, are you guys releasing any portable version soon? thank you.

**Comment by mpociot** [comment]
We don't have a plan for that. What's the reason for a portable version?

**Reply by ezequiel9** [reply]
It's because I want to use 2 different computers to develop. So I could put everything (the whole server like laragon does) into my portable ssd hard drive and plug it on any computer. If you ever do that would be great! Thanks.
